### PR TITLE
[Tram] The crossing lights actually indicate the danger of crossing

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request
We had a previous PR that increased damage, but it turns out that the traffic lights weren't changing fast enough for the new TG tram with the various character types we have.

## How This Contributes To The Skyrat Roleplay Experience
They don't get hit by the tram walking across on a green/amber light.

## Changelog

:cl: LT3
fix: The Tramstation crossing lights will now properly reflect if you're going to get smacked by it if you try to cross.
/:cl:

### CODE REVIEW

File: modular_skyrat/master_files/code/modules/industrial_lift/crossing_signal.dm [hunk 1]
```diff
@@ -0,0 +1,6 @@
+	/// Skyrat edit for people being slow to cross the tracks of the tram, due to character size or whatever else. Make the lights change faster with the tram.
```
Comment (Zonespace27) [inline modular_skyrat/master_files/code/modules/industrial_lift/crossing_signal.dm]: include the DME you goof

File: modular_skyrat/master_files/code/modules/industrial_lift/crossing_signal.dm [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+	/// Proximity threshold for amber warning (slow people may be in danger)
+	var/amber_distance_threshold = 90
```
Comment (Zonespace27) [inline modular_skyrat/master_files/code/modules/industrial_lift/crossing_signal.dm]: you're not setting these vars on anything

Comment (Zonespace27) [inline modular_skyrat/master_files/code/modules/industrial_lift/crossing_signal.dm]: for changing vars you don't need `var/`, only on creating one

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

**Comment by Elli-Skala** [comment]
> The crossing lights actually indicate the danger of crossing
So you're making it permanently red?

**Comment by Tastyfish** [comment]
I have no problem with it being adjusted, as I originally designed it for pre-doom-tram, as you said.

A couple of little things:
- the Amber distance should be at least 20 higher than the red or else it's not enough time to mentally seperate them, or depending on the mood of the fast processor, might not even show up.
- ideally in my opinion, amber should mark the time it's unsafe to cross walking, and then red and save to cross running. Otherwise runners will just ignore the traffic light and go while it's in the middle of red. 

See if 90/70 or 80/60 work.

**Comment by lessthnthree** [comment]
Yeah after sleeping on it, I am going to keep playing with it, Ideally I'd like it to be amber in central when the tram is stopped on east or west. I'll work on it more and take it upstream.

**Comment by lessthnthree** [comment]
60/35 ended up as the sweet spot, so doing that upstream.

**Pull request closed without merging**